### DISCUSSION TITLE
How to connect to cluster with kubectl from commandline instead in web UI?

### DISCUSSION THREAD

**Discussion by mesuutt** [kubesphere/kubesphere#3425] [Q&A]
Hi,

I rent a server from a local cloud provider and the server come with kubesphere installed. I can run kubectl commands from web ui of kubesphere from below, bot I want to connect from commandline of my laptop. 
<img width="533" alt="Screen Shot 2021-03-02 at 22 51 48" src="https://user-images.githubusercontent.com/823338/109706345-e2d17780-7ba9-11eb-98e0-7f66d7867ed9.png">

I downloaded kubeconfig from web ui but I cannot connect from my laptop because server ip in kubeconfig is private network ip.

<img width="317" alt="Screen Shot 2021-03-02 at 22 52 47" src="https://user-images.githubusercontent.com/823338/109706458-05639080-7baa-11eb-90cf-fb082ec8b7ae.png">

I could not find the port which I change in downloaded kubeconfig.  How can I find the port?

**Comment by LinuxSuRen** [comment]
The port is `6443` if you installed KubeSphere via [kubekey](https://github.com/kubesphere/kubekey/). Actually, KubeSphere builds upon the original Kubernetes. So, it should be the port of apiserver from Kubernets.
